metals/src/main/scala/scala/meta/internal/metals/ServerCommands.scala

```diff
@@ -231,6 +231,20 @@ object ServerCommands {
        |""".stripMargin,
   )
 
+  val JvmRunEnvironment = new ParametrizedCommand[String](
+    "jvm-run-environment",
+    "Get JVM run environment",
+    "Gets the required environment for running classes including classpath, jvm options and environment variables",
+    "[string], where the string the name of the build target.",
+  )
+
+  val JvmTestEnvironment = new ParametrizedCommand[String](
+    "jvm-run-environment",
+    "Get jvm run environment",
+    "Gets the required environment for testing classes including classpath, jvm options and environment variables",
+    "[string], where the string the name of the build target.",
+  )
+
```

Comment:
Could you leave a link to the https://build-server-protocol.github.io/docs/extensions/jvm.html#test-environment-request and run-environment?